app/views/hosts/console/vmrc.html.erb

```diff
@@ -7,7 +7,7 @@
   <p><%= _('To launch the VMRC console, you need the VMRC software installed.') %></p>
   <p><%= link_to(_('Download VMRC'), 'https://www.vmware.com/go/download-vmrc', :target => '_blank', :rel => 'noopener noreferrer') %></p>
   <div class="blank-slate-pf-main-action">
-    <%= link_to(_('Launch Console'), @console[:console_url], :class => 'btn btn-primary btn-lg', :onclick => "$(this).attr('disabled', 'disabled');") %>
+    <%= link_to(_('Launch Console'), @console[:console_url], :class => 'btn btn-primary btn-lg', :onclick => "const myelem=$(this); setTimeout(function() { myelem.attr('disabled', 'disabled'); }, 1);") %>
```

Comment:
```suggestion
    <%= link_to(_('Launch Console'),
                @console[:console_url],
                :class => 'btn btn-primary btn-lg',
                :data => {
                  :disable_with => "<i class='fa fa-spinner fa-spin'></i> #{_('Opening VMRC Console')}..."
                }) %>
```
It would result in:
![image](https://user-images.githubusercontent.com/2884324/78784003-d9915800-79a4-11ea-80c1-8753bffb1267.png)

Drawback is, that it's not easy to set a disabled style on btns disabled by `jquery-ujs`, it looks ok to me though.

Reply:
if it works (in firefox and chrome), I'm happy :)

Reply:
Yeah, it works OK in both. Will you update your PR?